ext_tables.php

```diff
@@ -92,6 +92,9 @@
     \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::allowTableOnStandardPages('tx_styleguide_inline_1n');
     \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::allowTableOnStandardPages('tx_styleguide_inline_1n_child');
 
+    \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::allowTableOnStandardPages('tx_styleguide_inline_1_1');
```

Comment:
if you rename the tables from "_1_1" to "_11" and "_1_1_child" to "_11_child", the auto-data generator should find them and should create some data for it (hopefully correct data, but that should have a look into the generator, then).